src/qibo/gates/gates.py

```diff
@@ -502,6 +529,11 @@ def __init__(self, *q, delay: int = 0):
         self.target_qubits = tuple(q)
 
 
+def is_clifford_given_angle(angle):
```

Comment:
```suggestion
def _is_clifford_given_angle(angle):
```
and everywhere else it appears